.github/workflows/custom_helpline.yml

```diff
@@ -0,0 +1,120 @@
+# This is a basic workflow to build and upload Webchat to S3 bucket with Github Actions
+
+name: Custom Helpline Release
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
```

Comment:
Could we make it so that we only pass 2 parameters to the workflow, helpline code & environment, and generate these 4 variables from those? Then we can make it so that this workflow can be called manually, which would be hard to do right now having to pass the 4 parameters.

This is for 4 reasons 

- it aligns it with the other 'custom helpline' workflows for flex & serverless
- it can be handy to call the custom workflow directly, for temporary test accounts we don't to check in a workflow for, or during account setup where the individual workflow doesn't exist yet
- It will make future 'deploy all staging' and 'deploy all prod' workflows much easier
- we'll probably move away from the 'workflow for every account' approach and just use this workflow for all deploys at some point, because what we do atm won't scale, it's already getting unwieldy

You would need to change the casing and translate production -> prod, staging -> stg & development -> dev for the bucket names but that should be doable

It could be done in a follow up PR to this but might be nice to get it done all at once, since it would involve changing all 25 workflows again

Reply:
> You would need to change the casing and translate production -> prod, staging -> stg & development -> dev for the bucket names but that should be doable

Instead of translating (which may not work because some build commands are `${shortCode}-prod` and some are `${shortCode}-production` could we, please, go ahead and standardize to the full environment name which is already part of [CHI-1564](https://tech-matters.atlassian.net/browse/CHI-1564)

Reply:
Oh that is for the chat bucket as well as the build command.

Oof. The lack of a single environment naming convention will continue to make life harder than it needs to be at every turn until it is resolved.

Reply:
@stephenhand, all feedback has been implemented.